packages/frontend/src/commons/chains/taiko.ts

```diff
@@ -0,0 +1,51 @@
+import { SupportedAmm } from "@metrom-xyz/sdk";
+import { SupportedChain, ADDRESS } from "@metrom-xyz/contracts";
+import { type ChainData } from "..";
+import { TaikoLogo } from "@/src/assets/logos/chains/taiko";
+import { PankoLogo } from "@/src/assets/logos/amms/panko";
+
+export const taikoData: ChainData = {
+    metromContract: ADDRESS[SupportedChain.Taiko],
+    icon: TaikoLogo,
+    amms: [
+        {
+            slug: SupportedAmm.Panko,
+            logo: PankoLogo,
+            name: "Kim",
```

Comment:
Here is should be `Panko`, no?

Reply:
Addressed in c74f1b7b7df6c5b951d4257a61666354c716b54a